src/OpenTelemetry/Metrics/BaseExportingMetricReader.cs

```diff
@@ -91,7 +91,18 @@ internal override bool ProcessMetrics(in Batch<Metric> metrics, int timeoutMilli
             // TODO: Do we need to consider timeout here?
             try
             {
-                return this.exporter.Export(metrics) == ExportResult.Success;
+                OpenTelemetrySdkEventSource.Log.MetricReaderEvent("BaseExportingMetricReader calling Exporter.Export method.");
+                var result = this.exporter.Export(metrics);
+                if (result == ExportResult.Success)
+                {
+                    OpenTelemetrySdkEventSource.Log.MetricReaderEvent("Exporter.Export succeeded.");
```

Comment:
I would suggest that we generate a string in the ctor with the actual Exporter info (e.g. type name, instance id). Doesn't have to be in this PR though.